### PR TITLE
fixed windows path issues

### PR DESCRIPTION
**Pull request by logan-anderson**
Fixed a few path issues on windows. In this state I am able to run tina-cloud-starter in the mono repo on my windows machine using powershell.

### CODE REVIEW

File: packages/@tinacms/graphql/src/primitives/database/bridge.ts [hunk 1]
```diff
@@ -26,14 +28,18 @@ export class FilesystemBridge implements Bridge {
     this.rootPath = rootPath || ''
   }
   public glob = async (pattern: string) => {
-    const items = await fg(path.join(this.rootPath, pattern, '**/*'), {
+    const basePath = path.join(this.rootPath, ...pattern.split('/'))
+    const items = await fg( path.join(basePath, '**','/*').replace(/\\/g, '/'), {
       dot: true,
     })
+    const posixRootPath = normalize(this.rootPath)
+    console.log({rootPath: this.rootPath})
```
Comment (logan-anderson) [inline packages/@tinacms/graphql/src/primitives/database/bridge.ts]: I should remove this

File: packages/@tinacms/graphql/src/primitives/database/bridge.ts [hunk 2]
```diff
@@ -26,14 +28,18 @@ export class FilesystemBridge implements Bridge {
...
     return items.map((item) => {
-      return item.replace(this.rootPath, '').replace(/^\/|\/$/g, '')
+      return item.replace(posixRootPath, '').replace(/^\/|\/$/g, '')
     })
   }
   public get = async (filepath: string) => {
+    console.log({filepath})
```
Comment (logan-anderson) [inline packages/@tinacms/graphql/src/primitives/database/bridge.ts]: remove this too

File: packages/@tinacms/graphql/src/primitives/schema/index.ts [hunk 3]
```diff
@@ -92,7 +92,7 @@ export class TinaSchema {
   }
   public getCollectionByFullPath = async (filepath: string) => {
     const collection = this.getCollections().find((collection) => {
-      return filepath.startsWith(collection.path)
+      return filepath.startsWith( collection.path)
```
Comment (logan-anderson) [inline packages/@tinacms/graphql/src/primitives/schema/index.ts]: Remove this space

**Comment by logan-anderson** [comment]
This has not been tested back in unix/linux/macos

File: packages/@tinacms/graphql/src/primitives/schema/index.ts [hunk 4]
```diff
@@ -108,7 +108,7 @@ export class TinaSchema {
   }> => {
     let template
     const collection = this.getCollections().find((collection) => {
-      return filepath.startsWith(collection.path)
+      return filepath.replace('\\','/').startsWith(collection.path)
```
Comment (Enigmatical) [inline packages/@tinacms/graphql/src/primitives/schema/index.ts:111]: Should we also use `normalize-path` here, as well?  Set `filepath.startsWith(collection.path)` to some variable and run `normalize-paths` against it?  Or is this solving _another_ Windows-related problem?

Comment (jeffsee55) [reply packages/@tinacms/graphql/src/primitives/schema/index.ts:111]: I'm not sure we need that, it's comparing `my/path` as a string

**Review by Enigmatical — APPROVED** [review]

**Pull request merged**